src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -118,44 +118,51 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AdjoinedNotFlagError" xml:space="preserve">
-    <value>Adjoined alias is not a flag</value>
+    <value>Adjoined alias is not a flag: '{0}'</value>

```

Comment:
>{0} [](http://example.com/codeflow?start=42&length=3)

Use {Locked="{0}","{1}"} in the comment across all the changes that added placeholders across this file to prevent human errors